addons/account/models/account_journal_dashboard.py

```diff
@@ -663,6 +713,29 @@ def action_create_vendor_bill(self):
             'views': [[view_id, 'form']],
         }
 
+    def action_open_payment_items(self):
+        view_id = self.env.ref('account.view_move_line_payment_tree').id
+        search_view_id = self.env.ref('account.view_account_move_line_payment_filter').id
+        return {
+            'name': _('Payment Items'),
+            'type': 'ir.actions.act_window',
+            'res_model': 'account.move.line',
+            'views': [[view_id, 'tree']],
+            'view_mode': 'tree',
+            'search_view_id': [search_view_id, 'search'],
+            'target': 'current',
+            'context': {'search_default_group_by_partner':1, 'expand': 1},
+            'domain': [('parent_state', '=', 'posted'), ('amount_residual', '!=', 0), ('account_id.reconcile', '=', True)],
```

Comment:
The point of putting those in the domain instead of using the already existing filters was for the search bar not to be filled with filters the user will never want to remove and therefore don't need to be displayed.
I couldn't figure a way to hide filters so I put those in the domain.